### PR TITLE
Loading state  for risk cell

### PR DESCRIPTION
**Pull request by aleksandr-tikhonov**
Hi,

This PR is attempt to fix incorrect behaviour of loading state for risk cell on the home screen. 
The idea of this PR is that move loading logic from different places in view controllers to one place - `RiskProvider`. And `RiskProvider` provides information about current status of loading, consumers can get this information by corresponding closure. Also this approach help us to reduce complexity of sync loading status between different screens. 

Looking forward your feedback.

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeUnknownRiskCellConfigurator.swift [hunk 1]
```diff
@@ -44,8 +44,9 @@ final class HomeUnknownRiskCellConfigurator: HomeRiskLevelCellConfigurator {
 
 	override func configure(cell: RiskLevelCollectionViewCell) {
 		cell.delegate = self
-
 		let title: String = isLoading ? AppStrings.Home.riskCardStatusCheckTitle : AppStrings.Home.riskCardUnknownTitle
+		
+		print(Unmanaged.passUnretained(self).toOpaque(), cell, title)
```
[the remark this answers is not in the document]
Comment (aleksandr-tikhonov) [reply src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeUnknownRiskCellConfigurator.swift]: I removed print code, thanks

**Comment by aleksandr-tikhonov** [comment]
PR is 99% ready, just need, after short conversation with @ChristianKienle, to make sure what to choose `.wait` or `.notify` for `group` in `RiskProvider`.

**Comment by aleksandr-tikhonov** [comment]
> PR is 99% ready, just need, after short conversation with @ChristianKienle, to make sure what to choose `.wait` or `.notify` for `group` in `RiskProvider`.

I reverted changes back to .wait approach.

File: src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift [hunk 2]
```diff
@@ -27,9 +27,16 @@ protocol ExposureSummaryProvider: AnyObject {
 }
 
 final class RiskProvider {
-	private let consumers = NSHashTable<RiskConsumer>.weakObjects()
-	private let queue = DispatchQueue(label: "com.sap.RiskLevelProvider")
+
+	private let queue = DispatchQueue(label: "com.sap.riskprovider.queue")
```
Comment (ChristianKienle) [inline src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift:31]:
```suggestion
	private let queue = DispatchQueue(label: "com.sap.RiskLevelProvider")
```

`.queue` is redundant and the naming convention usually is `$prefix.$className/structName`.

**Review by ChristianKienle — APPROVED** [review]
Approved. Consider the [(optional) suggestion](https://github.com/corona-warn-app/cwa-app-ios/pull/873/files#r454291392) I made:

File: src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift (hunk 2 above)
Comment (aleksandr-tikhonov) [reply src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift]: ok, I reverted labels

**Review by haosap — APPROVED** [review]

**Pull request merged**